test/fail_compilation/retscope.d

```diff
@@ -352,25 +352,13 @@ int* foo11(int* x) @safe { return x; }
 
 /******************************************/
 
-void escape15() @safe
-{
-    int arg;
-    const(void)*[1] argsAddresses;
-    argsAddresses[0] = // MUST be an array assignment
-        (ref arg)@trusted{ return cast(const void*) &arg; }(arg);
-}
-
-/******************************************/
 /*
 TEST_OUTPUT:
-// FIXME FIXME FIXME
-fail_compilation/retscope.d(1003): Error: escaping reference to local variable f
-// FIXME FIXME FIXME
 ---
+fail_compilation/retscope.d(365): Error: escaping reference to local variable f
 ---
 */
 
-#line 1000
```

Comment:
It rather seems like a good idea to make tests position independent, what's the problem with it?